src/box/lua/execute.c

```diff
@@ -346,7 +346,8 @@ lua_sql_bind_decode(struct lua_State *L, struct sql_bind *bind, int idx, int i)
 		 * will be poped from Lua stack.
```

Comment:
Please use `sql:` instead of `c:` in commit-message title. Also, I think it is better to be more specific - for example `sql: fix use-after-poison in variable binding`.

Reply:
Commit message replaced.